rtengine/rawimagesource.cc

```diff
@@ -7441,6 +7467,9 @@ void RawImageSource::getrgbloc(int begx, int begy, int yEn, int xEn, int cx, int
 {
 //    BENCHFUN
     //used by auto WB local to calculate red, green, blue in local region
+    greenloc(0, 0);
+    redloc(0, 0);
+    blueloc(0, 0);
```

Comment:
This is not necessary because the arrays will always be resized a few lines below.